cmds/proxy_server/main.go

```diff
@@ -126,8 +134,12 @@ func createServer(f flags, gitCommit string) (*http.Server, error) {
 	router := mux.NewRouter().StrictSlash(true)
 
 	// setup explorer
-	explorer.Setup(router, f.explorer, f.redis, gitCommit)
-	rmbproxy.Setup(router, f.substrate)
+	if err := explorer.Setup(router, f.redis, gitCommit, f.postgresHost, f.postgresPort, f.postgresDB, f.postgresUser, f.postgresPassword); err != nil {
```

Comment:
Why not passing a `postgress` connection string instead of separate parts. Or also a db connection pool instance even then the setup can be generic over the actual underlying db type. (not sure if this is well known in the rest of the code)